client_secret.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"7983846294-o9gd526qhf0nfae1qu0ad0ilf46i517b.apps.googleusercontent.com","project_id":"handy-flame-137623","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"SooeW0faDjaoBaA3O74oM5p3","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
It came from the sample code I used to test out how to work with Google Calendar API. It can be deleted.